src/styles.scss

```diff
@@ -215,3 +215,53 @@ mat-card.dashboard-widget {
 .mat-select-disabled .mat-select-value {
   color: black;
 }
+
+/* Replace the max java integer in the page size dropdown with the label 'All'  */
+mat-option[ng-reflect-value='2147483647'] > span {
+	display: none;
+}
+
+/* Replace the max java integer in the page size dropdown with the label 'All'  */
+mat-option[ng-reflect-value='2147483647']:last-child:after {
+	content: 'All' !important;
+	visibility: visible !important;
+	display: block !important;
+	position: absolute !important;
+	background-color: transparent !important;
+}
+
+.mat-slide-toggle-thumb {
```

Comment:
Does this have to be defined here or could it be moved to `entity-list.component.scss`?

Reply:
It is moved now to list-paginator.component.scss.